cmd/vic-machine/create/create.go

```diff
@@ -181,6 +182,12 @@ func (c *Create) Flags() []cli.Flag {
 		},
 
 		// volume
+		cli.StringFlag{
+			Name:        "default-volume-store, vsd",
```

Comment:
I did not realize that is what we wanted from the end of that ticket. We can also do that. I think having this flag is more user friendly, but we can get rid of it as well.